docs/reference/requirement-sets/powerpoint-preview-apis.md

```diff
@@ -0,0 +1,43 @@
+---
+title: PowerPoint JavaScript preview APIs
+description: 'Details about upcoming PowerPoint JavaScript APIs.'
+ms.date: 10/23/2020
+ms.prod: powerpoint
+localization_priority: Normal
+---
+
+# PowerPoint JavaScript preview APIs
+
+New PowerPoint JavaScript APIs are first introduced in "preview" and later become part of a specific, numbered requirement set after sufficient testing occurs and user feedback is acquired.
+
+The first table provides a concise summary of the APIs, while the subsequent table gives a detailed list.
+
+[!INCLUDE [Information about using preview APIs](../../includes/using-preview-apis-host.md)]
+
+| Feature area | Description | Relevant objects |
+|:--- |:--- |:--- |
+| Insert and Delete Slides | Allows the insertion of existing slides into the current presentation from another presentation, as well as the ability to delete sildes. | [Slide.delete](/javascript/api/powerpoint/powerpoint.slide#delete--), [Presentation.insertSlidesFromBase64](/javascript/api/powerpoint/powerpoint.presentation#insertslidesfrombase64-base64file--options-)|
+
+## API list
+
+The following table lists the PowerPoint JavaScript APIs currently in preview. To see a complete list of all PowerPoint JavaScript APIs (including preview APIs and previously released APIs), see [all PowerPoint JavaScript APIs](/javascript/api/powerpoint?view=powerpoint-js-preview&preserve-view=true).
+
+| Class | Fields | Description |
+|:---|:---|:---|
+|[InsertSlideOptions](/javascript/api/powerpoint/powerpoint.insertslideoptions)|[formatting](/javascript/api/powerpoint/powerpoint.insertslideoptions#formatting)|Specifies which formatting to use during slide insertion.|
+||[sourceSlideIds](/javascript/api/powerpoint/powerpoint.insertslideoptions#sourceslideids)|Specifies the slides from the source presentation that will be inserted into the current presentation. These slides are represented by their IDs which can be retrieved from a `Slide` object.|
+||[targetSlideId](/javascript/api/powerpoint/powerpoint.insertslideoptions#targetslideid)|Specifies where in the presentation the new slides will be inserted. The new slides will be inserted after the slide with the given slide ID.|
```

Comment:
Aside: In [reference docs](https://docs.microsoft.com/javascript/api/powerpoint/powerpoint.insertslideoptions?view=powerpoint-js-preview#targetslideid), maybe should say "non-existent" instead of "non-existing"...

Reply:
Good catch. I'll file a bug to get that changed in the source.